### PR TITLE
Merge Carthage pre-built fixes to develop

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Related PRs or issues

[AB#67897](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/67897)

### CODE REVIEW

File: AppCenterDistribute/AppCenterDistributeTests/MSDistributeAppDelegateTests.m [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import <AppCenterDistribute/AppCenterDistribute.h>
```
Comment (jaelim-ms) [inline AppCenterDistribute/AppCenterDistributeTests/MSDistributeAppDelegateTests.m]: Why is this using umbrella header?

Comment (MatkovIvan) [reply AppCenterDistribute/AppCenterDistributeTests/MSDistributeAppDelegateTests.m]: "Definition of 'MSDistribute' must be imported from module 'AppCenterDistribute.MSDistribute' before it is required"

Comment (jaelim-ms) [reply AppCenterDistribute/AppCenterDistributeTests/MSDistributeAppDelegateTests.m]:
```suggestion
#import "MSDistribute.h"
```

File: AppCenterDistribute/AppCenterDistributeTests/MSDistributeIngestionTests.m [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import <AppCenterDistribute/AppCenterDistribute.h>
```
Comment (jaelim-ms) [inline AppCenterDistribute/AppCenterDistributeTests/MSDistributeIngestionTests.m]:
```suggestion
#import "MSDistribute.h"
```

**Pull request merged**